src/api/spec/controllers/webui/feeds_controller_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rails_helper'
+
+RSpec.describe Webui::FeedsController do
+  let!(:project) { create(:project) }
+  let!(:commit) { create(:project_log_entry, project: project) }
+  let!(:old_commit) { create(:project_log_entry, project: project, datetime: 'Tue, 09 Feb 2015') }
+
+  describe "GET commits" do
+    it "assigns @commits" do
+      get(:commits, { project: project, format: 'atom'  })
+      expect(assigns(:commits)).to eq([commit])
+    end
+
+    it "assigns @project" do
+      get(:commits, { project: project, format: 'atom'  })
+      expect(assigns(:project)).to eq(project)
+    end
+
+    it "fails if project is not existent" do
+      expect do
+        get(:commits, { project: 'DoesNotExist', format: 'atom'  })
+      end.to raise_error ActiveRecord::RecordNotFound
```

Comment:
Guess we don't have a coding convention rule for this, but I would prefer {}.to here